gateway/rdb/src/main/kotlin/org/doorip/gateway/rdb/token/RefreshTokenJpaEntity.kt

```
@@ -0,0 +1,34 @@
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.time.LocalDateTime
+import org.doorip.domain.entity.UserId
+
+@Table(name = "refresh_token")
+@Entity
+internal class RefreshTokenJpaEntity {
+    @Id
+    @Column(name = "refresh_token", columnDefinition = "binary(45)", nullable = false)
+    lateinit var refreshToken: ByteArray
```

detekt (1.23.7)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)